Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    aemo (0.3.0.pre.rc2)
+    aemo (0.3.0)
```

Comment:
Minor version bump

Reply:
Since the change is more of a patch and not adding a new functionality to the module, should the new version be 0.3.1 a la https://semver.org/

Reply:
👍